js/DropDownTree/Checkbox.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: CheckBox with DropDownTree widget for Syncfusion Essential JS
-description: Describes about Checkbox functionalities in DropDownTree widget for Syncfusion Essential JS
+title: Checkbox with JavaScript DropDownTree Control | Syncfusion 
```

Comment:
Checkbox in JavaScript DropDownTree Control | Syncfusion

Reply:
Done